pyanaconda/modules/localization/installation.py

```diff
@@ -53,16 +55,50 @@ def name(self):
         return "Configure language"
 
     def run(self):
-        self._write_language_configuration(self._lang, self._sysroot)
+        """Run the installation task."""
+        lang = self._get_supported_language()
+        self._write_language_configuration(lang)
+
+    def _get_supported_language(self):
+        """Return a supported locale.
+
+        :return: a locale
+        """
+        if self._is_language_supported(self._lang):
+            return self._lang
+
+        log.debug("The '%s' locale is unsupported.", self._lang)
+        log.debug("Using the '%s' locale as a fallback.", self.LOCALE_FALLBACK)
+        return self.LOCALE_FALLBACK
+
+    def _is_language_supported(self, lang):
```

Comment:
I'm not sure if the name is specific enough. I went to check if this does not duplicate something from localization, but then realized this is specific to the new sysroot.

Reply:
Yep, I can rename it.

Reply:
Fixed.